galaxy/api/views/provider_namespace.py

```diff
@@ -42,6 +42,9 @@
 
 
 def check_provider_access(provider, user, name):
+    if user.is_staff:
+        return True
+
```

Comment:
check_provider_access() checks this itself but check_namespace_access() doesnt.
Any particular reason?

Nothing seems to use the return of this method, does it need to 'return True' or could it just 'return' ?